src/Confluent.Kafka/Consumer.cs

```diff
@@ -67,11 +67,11 @@ private void LogCallback(IntPtr rk, int level, string fac, string buf)
             if (OnLog == null)
             {
                 // A stderr logger is used by default if none is specified.
-                Loggers.ConsoleLogger(this, new LogMessage(name, level, fac, buf));
+                Loggers.ConsoleLogger(this, new LogMessage(name, SyslogLevel.Error, fac, buf));
```

Comment:
The level needs to be transferred all the way through from librdkafka, not replaced by error here. Also `int` is besty replaced by `SyslogLevel` all the way back to the p/invoke code (i.e. including in `LibRdKafka.LogDelegate`).

Reply:
I hope I understood you correct, you want me to change `LogDelegate` definition to use `SyslogLevel` and platform invoke code will map int level from native to `SyslogLevel` enumeration.